feat: miscellaneous enhancements to the bot

raisedadead opened the pull request:
- Have updated the configs and the bot will not cry about the closes keyword now.
- Added the commitizen and standard commits goodies.
- Updated the web view of the bot.

Closes #9

raisedadead committed:
refactor(webapp): update webview

raisedadead committed:
fix: Update test repo configs

raisedadead committed:
fix(close-keywords): remove messages about close keywords

raisedadead commented:
@BerkeleyTrue I have done some updates to the bot, and will be making more in future when I have more time at hand. 

For now, this can go in as it is. I have updated the PR OP to explain whats added for now.

Can you take a look and deploy if this is ok.

BerkeleyTrue commented on package.json:
```diff
@@ -6,12 +6,18 @@
     "start": "node run-server.js",
     "test": "./node_modules/.bin/babel-node node_modules/mocha/bin/_mocha --recursive -R spec",
-    "coverage": "./node_modules/.bin/babel-node ./node_modules/istanbul/lib/cli cover -- node_modules/mocha/bin/_mocha --recursive --reporter spec"
+    "coverage": "./node_modules/.bin/babel-node ./node_modules/istanbul/lib/cli cover -- node_modules/mocha/bin/_mocha --recursive --reporter spec",
```
You don't need to fully qualify the path of a local module. npm will automatically create a global symbol local bin files while running scripts

raisedadead committed:
test: Remove old test configs

This commit temporarily removes the old test and coverage configs and is
to be fixed via #2

raisedadead commented:
@BerkeleyTrue I have actually gone ahead and removed the old test and coverage configs since they aren't working right now, we already have an issue to track those, and I plan to work on them.

BerkeleyTrue commented:
👍 

Is this ready for review?

raisedadead commented:
Ping @BerkeleyTrue I guess we could merge this and deploy.
